test/amazonica/test/core.clj

```diff
@@ -23,6 +25,8 @@
              elastictranscoder
              glacier
              identitymanagement
+             iot
+             iotdata
```

Comment:
Not germane to the PR except in the sense that one of the nses I'm adding to is `iot`.

I could add code here to just require all of them programmatically, but I didn't want to cloud this PR.